collections/_tips/developing.zh-hans.md

```diff
@@ -172,8 +172,8 @@ footer: >
     <aside>
       <h3>相关文章</h3>
       <ul>
-        <li><a href="#">Bear receives key to city</a></li>
-        <li><a href="#">Superbear stands for mayor</a></li>
+        <li><a href="#">超级熊收到城市钥匙</a></li>
+        <li><a href="#">超级熊支持市长</a></li>
```

Comment:
@heybran In this context, "Superbear stands for mayor" means "Superbear is candidate to become mayor" rather than "Superbear supports the mayor".